src/core_plugins/tagcloud/public/tag_cloud_controller.html

```diff
@@ -2,6 +2,6 @@
   <div ng-show="vis.params.hideLabel" class="tagcloud-custom-label"></div>
   <div class="tagcloud-notifications">
     <div class="tagcloud-truncated-message">The number of tags has been truncated to avoid long draw times.</div>
-    <div class="tagcloud-incomplete-message">The container is too small to display the entire cloud. Tags may appear cropped or be ommitted.</div>
+    <div class="tagcloud-incomplete-message">The container is too small to display the entire cloud. Tags may appear cropped or be omitted.</div>
```

Comment:
The fix LGTM. I tripped a little over the "may appear" -- I'd probably go with "Tags might be cropped or omitted." But truthfully, that's just a style preference.